git_machete/cmd.py

```diff
@@ -424,6 +425,20 @@ def get_git_dir():
     return git_dir
 
 
+def get_abs_git_dir():
+    global abs_git_dir
+    if not abs_git_dir:
+        try:
+            # git_dir is absolute when we are in a directory of submodule
```

Comment:
I'm afraid we can't really rely on this behavior of git... I'm afraid this can silently change even b/w minor versions in future :/